src/components/WorkspaceFilter/WorkspaceFilter.js

```diff
@@ -18,142 +19,75 @@ const WorkspaceFilter = props => {
   const [show, setShow] = useState(false)
   const [filter, setFilter] = useState({
     // Main filters
-    fastWifi: false,
-    lotsOfSeats: false,
-    quiet: false,
-    // lively: false,
-    // outlets: false,
-    // openNow: false,
-    // openEarly: false,
-    // openLate: false,
-    cowork: false,
-    // library: false,
-    // cafe: false,
-    bool_food: false,
-    bool_coffee: false,
-    bool_alcohol: false,
+    fastWifi: 'off',
+    lotsOfSeats: 'off',
+    cafe: 'off',
+    library: 'off',
+    quiet: 'off',
+    bool_outlet: 'off',
+    bool_food: 'off',
+    bool_coffee: 'off',
+    bool_alcohol: 'off',
     // Secondary filters
-    bool_petfriendly: false,
-    bool_seating: false,
-    bool_parking: false,
-    bool_goodforgroup: false,
-    bool_outdoorSpace: false
+    bool_petfriendly: 'off',
+    bool_seating: 'off',
+    bool_parking: 'off',
+    bool_goodforgroup: 'off',
+    bool_outdoorSpace: 'off'

```

Comment:
I would prefer to use booleans `true` or `false` to maintain state. Is there an advantage to using `'on'` and `'off'`?